src/coreComponents/physicsSolvers/contact/SolidMechanicsLagrangeContact.cpp

```diff
@@ -2356,13 +2356,6 @@ bool SolidMechanicsLagrangeContact::isFractureAllInStickCondition( DomainPartiti
   return ( ( numNewSlip + numSlip + numOpen ) == 0 );
 }
 
-real64 SolidMechanicsLagrangeContact::setNextDt( real64 const & currentDt,
```

Comment:
Is this intentional? Do we really want this solver to inherit the behaviour of the base class?

Reply:
intentional - yes but I forgot about that change and didn't mention it in the description, sorry
it is very confusing what `SolidMechanicsLagrangeContact::setNextDt` does right now by returning `currentDt` and next time step, this basically blocks time step increase? (CC @CusiniM @matteofrigo5 @castelletto1)
could go in a small simple PR with just that fix

Reply:
@paveltomin Put this in its own PR.

Reply:
done https://github.com/GEOS-DEV/GEOS/pull/3490